index.md

```diff
@@ -12,35 +12,85 @@ First we set up & capture the content, then we render it in the scaffolding belo
 
 
 {% capture intro-left %}
-## ![Cockpit](/images/site/cockpit-logo.svg)
-{:.logo}
-
-<span aria-hidden="true" role="presentation">Cockpit</span> is a server manager that makes it easy to administer your GNU/Linux servers via a web browser.
-
+Cockpit makes it easy to administer your GNU/Linux servers via a web browser.
 {% endcapture %}
 
 {% capture intro-right %}
 [![Docker screenshot]({{ site.baseurl }}/images/site/screenshot-docker.png)]({{ site.baseurl }}/images/site/screenshot-docker.png){:.screenshot.zoom}
 {% endcapture %}
 
 
+{% assign blurbs_highlight = 6 %}
+
 {% capture blurbs %}
 
-[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png){:.screenshot.zoom}
-### Easy to use
+## Easy to use
+
+### Discoverable
 Cockpit makes Linux discoverable, allowing sysadmins to easily perform tasks such as starting containers, storage administration, network configuration, inspecting logs and so on.
 
+[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png)
+{:.screenshot.zoom}
+
+### Designed & tested
+Cockpit has been designed with your goals in mind. We also routinely test Cockpit with usability studies to make it work the way you'd expect. As a result, Cockpit gets easier to use all the time.
+
+### Team-friendly
+Cockpit is useful for seasoned admins and is friendly enough for those new to Linux. We've designed Cockpit to be friendly for whole teams.
+
+### Packages included
+[Installing and running Cockpit]({{ site.baseurl }}/running.html) is simple. It's already included in most of the major distributions.
+
 ---
 
-[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png){:.screenshot.zoom}
-### No interference
+## Integrated
+
+### Plays well with others
 Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
 
+[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png)
+{:.screenshot.zoom}
+
+### Sign in like normal
+By default, Cockpit uses [system credentials]({{ site.baseurl }}/guide/latest/privileges). You don't need to set up any special accounts. Network-wide logins are also supported through [single-sign-on]({{ site.baseurl }}/guide/latest/sso) and other [authentication]({{ site.baseurl }}/guide/latest/authentication) techniques.
+
+### Self-contained
+Cockpit is self-contained and uses existing system resources. You don't have to worry about setting up a webserver just to use Cockpit.
+
+### Uses existing APIs
+Cockpit uses APIs that already exist on the system. It doesn't reinvent subsystems or add a layer of its own tooling.
```

Comment:
I'd add a rationale for this - "It can be used in parallel with other administration tools like the command line or Ansible."

Reply:
I think that's more "plays well with others", right? We could move sections around if you think it's more related.

(I'm revising the text of "plays well" section to accommodate this a bit better.)